src/layers/TrajservLayer.test.js

```diff
@@ -0,0 +1,36 @@
+import 'jest-canvas-mock';
+// import Map from 'ol/Map';
```

Comment:
```suggestion
```

Reply:
Some comments to be removed in this test file.

Reply:
removed